atlas/portal/README.md

```diff
@@ -0,0 +1,53 @@
+# portal
```

Comment:
Here and in other readme's. This is just a template, which should be changed (ideally) by developer before the PR opening. It's not critical in our tasks, but please keep in mind, that we have to provide all the required/useful information in the README.md.